includes/sanitizers/class-amp-core-theme-sanitizer.php

```diff
@@ -1023,6 +1023,18 @@ static function() use ( $args ) {
 						right: calc(100% + 2rem);
 					}
 
+					.admin-bar .menu-modal {
+						/* Use padding to shift down modal because amp-lightbox has top:0 !important. */
+						padding-top: 32px;
```

Comment:
Oh, I just noticed I stored this in my browser, but not in the PR...
Have to watch out for this next time, I thought it was fixed.

Reply:
Oh, how did you store it in your browser?

Reply:
@westonruter See https://developers.google.com/web/updates/2018/01/devtools#overrides